app/controllers/api/account.php

```diff
@@ -1498,6 +1498,13 @@ function sendSessionAlert(Locale $locale, Document $user, Document $project, Doc
                         'providerType' => MESSAGE_TYPE_EMAIL,
                         'identifier' => $email,
                     ]));
+
+                    $queueForEvents
+                        ->setEvent('users.[userId].create')
```

Comment:
Is this going to mess up the event for the endpoint itself? since this replaces the event set during init, right?

https://github.com/appwrite/appwrite/blob/fdd39a29c3667069d97af3fc40bd5098bd32c41c/app/controllers/shared/api.php#L437